_docs/development/build-instructions-osx.md

```diff
@@ -1,7 +1,7 @@
 ---
 version: v1.0.1
 category: Development
-title: 'Build Instructions Osx'
+title: 'Build Instructions OS X'
```

Comment:
This page is regenerated automatically so it needs to be fixed in here: https://github.com/electron/electron.atom.io/blob/2c099dd20c40765d7e1fde3c289f359224844a4f/lib/fetch-docs.js#L170